spec/views/content_items/index.html.erb_spec.rb

```diff
@@ -5,24 +5,31 @@
 
   before { assign(:content_items, content_items) }
 
-  it 'render the table header' do
+  it 'renders the table header with the right headings' do
     render
 
-    expect(rendered).to have_selector('table thead', text: 'Content Ids')
+    expect(rendered).to have_selector('table thead tr:first-child', text: 'Content Ids')
+    expect(rendered).to have_selector('table thead tr:nth(1)', text: 'Content URL')
   end
 
   it 'renders a row per Content Item' do
     render
 
-    expect(rendered).to have_selector('table tbody tr', count: 2)
+    expect(rendered).to have_selector('table tbody tr', count: 4)
```

Comment:
This is meant be 4 rows.. becuase in the template I created an extra table row have.. I have changed it to only two table rows and 2 cells per row